src/cpp/server/server_cc.cc

```diff
@@ -77,14 +77,8 @@ class DefaultGlobalCallbacks final : public Server::GlobalCallbacks {
   void PostSynchronousRequest(ServerContext* context) override {}
 };
 
-std::shared_ptr<Server::GlobalCallbacks> g_callbacks = nullptr;
-gpr_once g_once_init_callbacks = GPR_ONCE_INIT;
-
-void InitGlobalCallbacks() {
-  if (!g_callbacks) {
-    g_callbacks.reset(new DefaultGlobalCallbacks());
-  }
-}
+DefaultGlobalCallbacks g_default_callbacks;
```

Comment:
I agree that the current approach has a problem, but this variant also has a problem. In particular, neither of the objects in question allows constexpr initialization or trivial destruction, both of which should be needed for global objects.

Reply:
Thanks for your review. I'll apply c21393e553e7ab528edb5966c835bc0832ea4edf to match ClientContext's GlobalCallbacks.

Reply:
I've fixed this.